docs/index.md

```diff
@@ -279,19 +279,40 @@ If you want to add another cluster (like asia-east1 in the above screenshots), r
    spec:
      # clusterName is required and must be unique among all managed clusters
      clusterName: $SHORT_CLUSTERNAME
-     git:
-       syncRepo: git@$GITLAB_HOSTNAME:platform-admins/anthos-config-management.git
-       syncBranch: master
-       secretType: ssh
+     enableMultiRepo: true
+     enableLegacyFields: false
      policyController:
        enabled: true
    EOF
    ```
+
+1. Create a Root Sync configuration for your cluster
+
+   ```shell
+   export NEW_REGION="asia-east1"
+   export SHORT_CLUSTERNAME="prod-${NEW_REGION}"
+   cat > root-sync.yaml <<EOF
+   apiVersion: configsync.gke.io/v1alpha1
+   kind: RootSync
+   metadata:
+     name: root-sync
+     namespace: config-management-system
+   spec:
+     sourceFormat: hierarchy
+     git:
```

Comment:
Suggest adding brackets for GITLAB_HOSTNAME: `${GITLAB_HOSTNAME}`.
Sorry for not catching it earlier

Reply:
Done